silverback/application.py

```diff
@@ -52,7 +60,8 @@ def __init__(self, settings: Optional[Settings] = None):
         logger.info(f"Loading Silverback App with settings:\n  {settings_str}")
 
         self.broker = settings.get_broker()
-        self.contract_events: Dict[AddressType, Dict[str, ContractEvent]] = {}
+        # NOTE: If no tasks registered yet, defaults to empty list instead of raising KeyError
+        self.tasks: defaultdict[TaskType, list[TaskData]] = defaultdict(list)
```

Comment:
oh man, I forgot all about `defualtdict`!  A pattern I keep working around with dumb solutions.  Thanks for the reminder.